docs/docs/cmd/spo/tenant/tenant-settings-set.mdx

```diff
@@ -14,257 +14,494 @@ m365 spo tenant settings set [options]
-: Enables or disables notifications in SharePoint. Allowed values `true`, `false`
+: Enables or disables notifications in SharePoint. Allowed values `true`, `false`.
 
 `--OwnerAnonymousNotification [OwnerAnonymousNotification]`
-: Enables or disables owner-anonymous notification. Allowed values `true`, `false`
+: Enables or disables owner-anonymous notification. Allowed values `true`, `false`.
 
 `--CommentsOnSitePagesDisabled [CommentsOnSitePagesDisabled]`
-: Enables or disables comments on site pages. Allowed values `true`, `false`
+: Enables or disables comments on site pages. Allowed values `true`, `false`.
 
 `--SocialBarOnSitePagesDisabled [SocialBarOnSitePagesDisabled]`
-: Enables or disables social bar on site pages. Allowed values `true`, `false`
+: Enables or disables social bar on site pages. Allowed values `true`, `false`.
 
 `--OrphanedPersonalSitesRetentionPeriod [OrphanedPersonalSitesRetentionPeriod]`
-: Specifies the number of days after a user's Active Directory account is deleted that their OneDrive for Business content will be deleted. The value range is in days, between 30 and 3650. The default value is 30
+: Specifies the number of days after a user's Active Directory account is deleted that their OneDrive for Business content will be deleted. The value range is in days, between 30 and 3650. The default value is 30.
+
+`--CoreRequestFilesLinkExpirationInDays [CoreRequestFilesLinkExpirationInDays]`
+: Specifies the number of days before a Request files link expires for all SharePoint sites (not including OneDrive sites). The value can be from 0 to 730 days. To remove the expiration requirement, set the value to zero (0).
+
+`--OneDriveRequestFilesLinkExpirationInDays [OneDriveRequestFilesLinkExpirationInDays]`
+: Specifies the number of days before a Request files link expires for all OneDrive sites. The value can be from 0 to 730 days. To remove the expiration requirement, set the value to zero (0).
+
+`--ExternalUserExpirationRequired [ExternalUserExpirationRequired]`
+: Specifies whether to enable the external user expiration policy, where external users will be expired and removed from the site collection in a given number of days. Allowed values `true`, `false`.
+
+`--ExternalUserExpireInDays [ExternalUserExpireInDays]`
+: Specifies the number of days before an external user will expire and be removed from the site collection if the policy is enabled. Value can be from 30 to 730 days.
+
+`--ReduceTempTokenLifetimeEnabled [ReduceTempTokenLifetimeEnabled]`
+: Enables reduced session timeout for temporary URLs used by apps for document download scenarios. Reduction occurs when an app redeeming an IP address does not match the original requesting IP. Allowed values `true`, `false`.
+
+`--ReduceTempTokenLifetimeValue [ReduceTempTokenLifetimeValue]`
+: Sets the session timeout value for temporary URLs. The value can be set between 5 and 15 minutes and the default value is 15 minutes.
+.
```

Comment:
This period doesn't belong here.

Reply:
I found this period from this command: https://learn.microsoft.com/en-us/powershell/module/sharepoint-online/set-spotenant?view=sharepoint-ps#-reducetemptokenlifetimevalue. Can you check and confirm once? Should I remove it from description or keep it?

Reply:
I'm talking about the period character on line 243 😊

Reply:
Thanks, pushed new commit with required changes!